packages/experiments/src/components/signals/Signals.tsx

```diff
@@ -19,7 +19,7 @@ export const YouCheckedOutSignal: Signal = (props: ISignalProps): JSX.Element =>
     <Icon
       ariaLabel={ props.ariaLabel }
       className={ css(SignalStyles.signal, SignalsStyles.youCheckedOutl) }
-      iconName=''
+      iconName='checkedoutbyyou12'
     /> // TODO get correct icon
```

Comment:
remove comments

Reply:
also could you double check the <SomeoneCheckedOutSignal>, the icon is 'navigateForwad', is that right? There is an icon of CheckedOutByOther12

Reply:
I just went over the list with the designer and re-updated. Should be all good now.